### DISCUSSION TITLE
Show 16-bit icons in Explorer

### DISCUSSION THREAD

**Discussion by Anixx** [otya128/winevdm#1315] [General]
This page http://www.columbia.edu/~em36/otvdm.html tells: "The January 2023 version of the installer offers to run a small program (created by the author of otvdm) that allows icons of 16-bit programs to appear in Windows Explorer, instead of appearing with generic icons. I strongly recommend that you accept the option to run this program. It seems to have no ill effects, only benefits."

I wonder what this program is, and where can I find it stand-alone? Is there a repo?

**Comment by cracyc** [comment]
https://github.com/otya128/Icon16bitFix

**Comment by Anixx** [comment]
Should it be resident or launched only once?

**Comment by emendelson** [comment]
Read the "How to use" paragraph on the linked page. Run it once.

**Comment by Anixx** [comment]
How does it work? Does it change system files? Registry?

**Comment by cracyc** [comment]
It hooks USER32!PrivateExtractIconsW in explorer so it will load icons from an NE exe file.

**Comment by Anixx** [comment]
So, it is resident? Need to run it each time or only once? If it is resident, one has to run it each time.

**Comment by Anixx** [comment]
P.S. Works great.

**Comment by Anixx** [comment]
Would it be possible to have a Windhawk mod for this?
